scripts/presigned_url.yml

```diff
@@ -0,0 +1,27 @@
+version: '3.3'
```

Comment:
Is there a reason we need a separate service for this? I would say its maybe nicer to just add these to the existing `minio_s3.yml` ?

Reply:
I explain in the comment below